cloudigrade/account/migrations/0007_add_image_properties.py

```diff
@@ -0,0 +1,62 @@
+# Generated by Django 2.0.5 on 2018-06-01 10:14
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def migrate_is_windows_to_property(apps, schema_editor):
+    """Copy old is_windows value to an image tag."""
+
+    # Create image tags
+    ImageTag = apps.get_model('account', 'ImageTag')
+    windows_tag = ImageTag.objects.create(description='windows')
+    ImageTag.objects.create(description='rhel')
+
+    # Get current machine images
+    MachineImage = apps.get_model('account', 'MachineImage')
+    all_images = MachineImage.objects.filter(is_windows=True)
```

Comment:
This is a minor nitpick, but for readability I think `all_images` here should be renamed to something like `windows_images`. When I first looked at this function and tried to understand what it was doing, my lizard brain kept getting caught on "all" and I thought you were putting the windows tags on every image until I carefully reread each line.

Reply:
👍